calculator.rb

```diff
@@ -0,0 +1,131 @@
+
+def newline
+  puts "\n"
+end
+
+newline
+puts "Welcome to the calculator program! Here are the operators you can use:"
+puts "  add (+)"
+puts "  subtract (-)"
+puts "  multiply (*)"
+puts "  divide (/)"
+puts "  exponify (**)"
+
+print "\nWhich operator would you like to use? Enter the name or symbol: "
+command = gets.chomp.downcase
+
+print "\nWhat is your first number for this operation? "
+first_num = gets.chomp.downcase
+
+# Tried to create a custom method but I couldn't get it to work :(
+# Learned in class: "puts" in a method is a no-no
+
+# def string_thing(string)
+#   if string.empty?
+#     return puts "\nSorry, that is not a valid number. Exiting program."
+#   else string = string.to_f
+#   end
+# end
+
+# Tried to create an array of letters that I could use as object
+# in include? method (after converting input to an array of single letters)
+# but couldn't make that work either. 
+# Include? works fine with a literal but maybe it doesn't work with
+# an array as object?
+
+# first_num_array = first_num.split(//)
+# puts first_num_array
+
+# letters = Array ('a'..'z') 
+# # puts letters
+
+# if first_num.empty? 
+#   return puts "\nOops, that is not a valid number. Exiting program."  
+# elsif first_num_array.include?(letters)
+#   return puts "\nOops, that is not a valid number. Exiting program."    
+# else
+#   first_num = first_num.to_f  
+# end
+
+# Another failed attempt ... 
+
+# def validate(number)
+#   if number.empty? 
+#     return puts "\nOops, that is not a valid number. Exiting program."  
+#   elsif number.include?("a") 
+#     return puts "\nOops, that is not a valid number. Exiting program."   
+#   elsif number.include?("b") 
```

Comment:
When you go to turn work in, please remove "dead code". It's good if you want to document your process, but as we head into week 2 we will show you tools that will track the history of a file for you.